eks_audit_logs.tf

```diff
@@ -0,0 +1,106 @@
+data "aws_cloudwatch_log_groups" "eks" {
+  count                 = var.enable_eks_audit_logs_pipeline ? 1 : 0
+  log_group_name_prefix = "/aws/eks/"
+}
+
+resource "aws_kinesis_firehose_delivery_stream" "firehose" {
+  count       = var.enable_eks_audit_logs_pipeline ? 1 : 0
+  name        = "ksoc-audit-logs"
+  destination = "extended_s3"
+
+  extended_s3_configuration {
+    role_arn   = aws_iam_role.firehose[0].arn
+    bucket_arn = aws_s3_bucket.audit_logs[0].arn
+
+    buffering_interval = 60
+    buffering_size     = 5 // MiB
+  }
+}
+
+resource "aws_s3_bucket" "audit_logs" {
+  count         = var.enable_eks_audit_logs_pipeline ? 1 : 0
+  bucket        = "ksoc-connect-eks-audit-logs"
+  force_destroy = true
```

Comment:
Please add additional tags to the bucket. We should be able to automatically create sync-task based on the new bucket fetch by cloud query